### PR TITLE
[snapchat-gradle] make sure PR test (example) uses locally built gradle

### PR DESCRIPTION
**Pull request by ericcrahensc**

### CODE REVIEW

**Comment by snapenginesc** [comment]
Building **snapchat-gradle** ([#15](https://snapengine-builder.sc-corp.net/jenkins/job/snapchat-gradle/15/))  /cc  @Snapchat/seattle-infrastructure

**Comment by snapenginesc** [comment]
Failed to build **snapchat-gradle** ([#15](https://snapengine-builder.sc-corp.net/jenkins/job/snapchat-gradle/15/))

**Comment by snapenginesc** [comment]
Building **snapchat-gradle** ([#16](https://snapengine-builder.sc-corp.net/jenkins/job/snapchat-gradle/16/))  /cc  @Snapchat/seattle-infrastructure

File: example/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://storage.googleapis.com/snapengine-builder/gradle-3.0-snapchat.zip
+distributionUrl=file\://../build/gradle-3.0-snapchat.zip
```
Comment (faizalkassamalisc) [inline example/gradle/wrapper/gradle-wrapper.properties]: nice

**Comment by snapenginesc** [comment]
Failed to build **snapchat-gradle** ([#16](https://snapengine-builder.sc-corp.net/jenkins/job/snapchat-gradle/16/))

**Comment by snapenginesc** [comment]
Building **snapchat-gradle** ([#17](https://snapengine-builder.sc-corp.net/jenkins/job/snapchat-gradle/17/))  /cc  @Snapchat/seattle-infrastructure

**Comment by snapenginesc** [comment]
Failed to build **snapchat-gradle** ([#17](https://snapengine-builder.sc-corp.net/jenkins/job/snapchat-gradle/17/))

**Comment by snapenginesc** [comment]
Building **snapchat-gradle** ([#18](https://snapengine-builder.sc-corp.net/jenkins/job/snapchat-gradle/18/))  /cc  @Snapchat/seattle-infrastructure

**Comment by snapenginesc** [comment]
Failed to build **snapchat-gradle** ([#18](https://snapengine-builder.sc-corp.net/jenkins/job/snapchat-gradle/18/))

**Comment by snapenginesc** [comment]
Building **snapchat-gradle** ([#19](https://snapengine-builder.sc-corp.net/jenkins/job/snapchat-gradle/19/))  /cc  @Snapchat/seattle-infrastructure

**Comment by snapenginesc** [comment]
Failed to build **snapchat-gradle** ([#19](https://snapengine-builder.sc-corp.net/jenkins/job/snapchat-gradle/19/))

**Comment by snapenginesc** [comment]
Building **snapchat-gradle** ([#20](https://snapengine-builder.sc-corp.net/jenkins/job/snapchat-gradle/20/))  /cc  @Snapchat/seattle-infrastructure

**Comment by snapenginesc** [comment]
Failed to build **snapchat-gradle** ([#20](https://snapengine-builder.sc-corp.net/jenkins/job/snapchat-gradle/20/))

**Comment by snapenginesc** [comment]
Building **snapchat-gradle** ([#21](https://snapengine-builder.sc-corp.net/jenkins/job/snapchat-gradle/21/))  /cc  @Snapchat/seattle-infrastructure

**Comment by snapenginesc** [comment]
Failed to build **snapchat-gradle** ([#21](https://snapengine-builder.sc-corp.net/jenkins/job/snapchat-gradle/21/))

**Comment by snapenginesc** [comment]
Building **snapchat-gradle** ([#22](https://snapengine-builder.sc-corp.net/jenkins/job/snapchat-gradle/22/))  /cc  @Snapchat/seattle-infrastructure

**Comment by snapenginesc** [comment]
Failed to build **snapchat-gradle** ([#22](https://snapengine-builder.sc-corp.net/jenkins/job/snapchat-gradle/22/))

**Comment by snapenginesc** [comment]
Building **snapchat-gradle** ([#23](https://snapengine-builder.sc-corp.net/jenkins/job/snapchat-gradle/23/))  /cc  @Snapchat/seattle-infrastructure

**Comment by snapenginesc** [comment]
Failed to build **snapchat-gradle** ([#23](https://snapengine-builder.sc-corp.net/jenkins/job/snapchat-gradle/23/))

**Comment by snapenginesc** [comment]
Building **snapchat-gradle** ([#24](https://snapengine-builder.sc-corp.net/jenkins/job/snapchat-gradle/24/))  /cc  @Snapchat/seattle-infrastructure

**Comment by snapenginesc** [comment]
Failed to build **snapchat-gradle** ([#24](https://snapengine-builder.sc-corp.net/jenkins/job/snapchat-gradle/24/))

**Comment by snapenginesc** [comment]
Building **snapchat-gradle** ([#25](https://snapengine-builder.sc-corp.net/jenkins/job/snapchat-gradle/25/))  /cc  @Snapchat/seattle-infrastructure

**Comment by snapenginesc** [comment]
Failed to build **snapchat-gradle** ([#25](https://snapengine-builder.sc-corp.net/jenkins/job/snapchat-gradle/25/))

**Comment by snapenginesc** [comment]
Building **snapchat-gradle** ([#26](https://snapengine-builder.sc-corp.net/jenkins/job/snapchat-gradle/26/))  /cc  @Snapchat/seattle-infrastructure

**Comment by snapenginesc** [comment]
Failed to build **snapchat-gradle** ([#26](https://snapengine-builder.sc-corp.net/jenkins/job/snapchat-gradle/26/))

**Comment by snapenginesc** [comment]
Building **snapchat-gradle** ([#27](https://snapengine-builder.sc-corp.net/jenkins/job/snapchat-gradle/27/))  /cc  @Snapchat/seattle-infrastructure

**Comment by snapenginesc** [comment]
Built **snapchat-gradle** sucessfully, **LGTM**! ([#27](https://snapengine-builder.sc-corp.net/jenkins/job/snapchat-gradle/27/))

**Comment by snapenginesc** [comment]
Building **snapchat-gradle** ([#28](https://snapengine-builder.sc-corp.net/jenkins/job/snapchat-gradle/28/))  /cc  @Snapchat/seattle-infrastructure

**Comment by snapenginesc** [comment]
Built **snapchat-gradle** sucessfully, **LGTM**! ([#28](https://snapengine-builder.sc-corp.net/jenkins/job/snapchat-gradle/28/))

**Pull request merged**